automatic/virtualbox_5.0.x/virtualbox_5.0.x.nuspec

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>virtualbox</id>
```

Comment:
no it's not.
since the 5.0.x branch is still officially supported, they are still pushing out bug fixes for it.
In this case there is no need for a new package, just a package to update when there is a bug release.
And in this case we just push it to the virtualbox package.